app/report/residency_fr_taxes_fr/ReportResidencyFrTaxesFr.tsx

```diff
@@ -0,0 +1,435 @@
+import { useMemo, useState } from "react";
+import Image from "next/image";
+import { EtradeGainAndLossesFileInput } from "@/app/guide/shared/EtradeGainAndLossesFileInput";
+import { useExchangeRates } from "@/hooks/use-fetch-exr";
+import { Button } from "@/app/guide/shared/ui/Button";
+import { GainAndLossEvent } from "@/lib/etrade/etrade.types";
+import {
+  FrTaxes,
+  applyFrTaxes,
+  getEmptyTaxes,
+} from "@/lib/taxes/taxes-rules-fr";
+import { Section } from "@/app/guide/shared/ui/Section";
+import {
+  isEspp,
+  isFrQualifiedRsu,
+  isFrQualifiedSo,
+  isUsQualifiedRsu,
+  isUsQualifiedSo,
+} from "@/lib/etrade/filters";
+import { Tooltip } from "@/components/ui/Tooltip";
+import { MessageBox } from "@/components/ui/MessageBox";
+import { Drawer } from "@/components/ui/Drawer";
+import { TaxableEventFr } from "@/components/TaxableEventFr";
+import { TaxableEventFr as TaxableEventFrProps } from "@/lib/taxes/taxable-event-fr";
+import {
+  ChevronDoubleLeftIcon,
+  ChevronDoubleRightIcon,
+  InformationCircleIcon,
+} from "@heroicons/react/24/solid";
+import { useFetchSymbolDaily } from "@/hooks/use-fetch-symbol-daily";
+
+export interface ReportResidencyFrTaxesFrProps {}
+
+export const ReportResidencyFrTaxesFr: React.FunctionComponent<
+  ReportResidencyFrTaxesFrProps
+> = () => {
+  const [isPrintMode, setIsPrintMode] = useState(false);
+  const [gainsAndLosses, setGainsAndLosses] = useState<GainAndLossEvent[]>([]);
+  const { values: rates, isFetching: isFetchingExr } = useExchangeRates(
+    gainsAndLosses.flatMap((event) => [event.dateSold, event.dateAcquired]),
+  );
+  const { data: ddogPrices, isFetching: isFetchingDDOG } =
+    useFetchSymbolDaily("DDOG");
```

Comment:
should we add a top level section to choose a symbol? even if the only available option is `DDOG` for now